src/components/VSelect/VSelect.js

```diff
@@ -116,6 +116,9 @@ export default {
   },
 
   methods: {
+    needsTile (tile) {
+      return !/v-list-tile$/.test(tile.tag)
```

Comment:
Doing some research on this, it appears that a comparison operator is [indefinitely](https://jsperf.com/matchgroup-vs-equal) faster than running .test. Since we have no direct need to compare an expression, can we change this to an absolute **equal**?

Reply:
Definitely, for `tile.tag` we can not seriously change this check, because by my tests it looks like `vue-component-40-v-list-tile`. On the other hand, we can access the property `tile.componentOptions.tag` and use the comparison operator:
`tile.componentOptions.tag === 'v-list-tile'`. However, I do not know whether it is legal

Reply:
I sure hope it's legal, Vuetify has been using componentOptions.tag in various components for quite a long time.

Reply:
Use name instead of tag, there was a PR a while ago to do that because someone was trying to rename components and tag caused some to not function correctly.

Reply:
@KaelWD i cant find the `name` property in the instance of `v-list-tile`

Reply:
It's in `$options`: #2312

I'm not sure if that issue will affect this actually, but it might be worth looking into.